SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x   | :x:                |
+| 3.x   | :white_check_mark: |
+| 4.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+At Verdaccio, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our team:
+
+* Report it either [Snyk Security Team](https://snyk.io/vulnerability-disclosure/) or [npmjs Security Team](https://www.npmjs.com/advisories/report?package=verdaccio), they will be in contact with us in case of confirming the vulnerability.
+* E-mail your findings to sing [verdaccio@pm.me](mailto:verdaccio@pm.me). If the report contains highly sensitive information, please consider encrypting your findings using our [PGP key](https://verdaccio.nyc3.digitaloceanspaces.com/gpg/publickey.verdaccio@pm.me.asc).
```

Comment:
to sing [verdaccio@pm.me](mailto:verdaccio@pm.me) => to [verdaccio@pm.me](mailto:verdaccio@pm.me)

Rest is perfect =)